simple_history/models.py

```diff
@@ -178,20 +196,34 @@ def contribute_to_class(self, cls, name):
             warnings.warn(msg, UserWarning)
 
     def add_extra_methods(self, cls):
-        def save_without_historical_record(self, *args, **kwargs):
+        def get_instance_predicate(
+            self: models.Model,
+        ) -> Callable[[models.Model], bool]:
+            def predicate(instance: models.Model) -> bool:
+                return instance == self
+
+            return predicate
+
+        def save_without_historical_record(self: models.Model, *args, **kwargs):
             """
             Save the model instance without creating a historical record.
 
             Make sure you know what you're doing before using this method.
             """
-            self.skip_history_when_saving = True
-            try:
-                ret = self.save(*args, **kwargs)
-            finally:
-                del self.skip_history_when_saving
-            return ret
+            with utils.disable_history(instance_predicate=get_instance_predicate(self)):
```

Comment:
If the instance being saved has cascading receivers configured on `post_save`, those would likely generate history here while the previous version they wouldn't.

I suspect this manager should disable all history rather than trying to only disable it for this instance.